src/Bicep.Core/Semantics/Namespaces/AzNamespaceType.cs

```diff
@@ -446,7 +446,12 @@ public static NamespaceType Create(string aliasName, ResourceScope resourceScope
         {
             return new NamespaceType(
                 aliasName,
-                Settings,
```

Comment:
Do we need to remove `Settings` since it's not used anymore?

Reply:
I am not sure I follow, its still being used AFAICT, can you elaborate?

Reply:
Scratch this - sorry I didn't realize it's still used in BicepCompletionProvider.